src/ert/gui/main.py

```diff
@@ -276,6 +276,7 @@ def _setup_suggester(
     suggest_layout = QVBoxLayout()
     buttons_layout = QHBoxLayout()
 
+    print(errors)
```

Comment:
This looks like it is just a debugging print.

Reply:
Oops, fixed